spec/controllers/api/web/push_subscriptions_controller_spec.rb

```diff
@@ -15,6 +15,7 @@
           p256dh: 'BEm_a0bdPDhf0SOsrnB2-ategf1hHoCnpXgQsFj5JCkcoMrMt2WHoPfEYOYPzOIs9mZE8ZUaD7VA5vouy0kEkr8=',
           auth: 'eH_C8rq2raXqlcBVDa1gLg==',
         },
+        standard: 1,
```

Comment:
Docs say this should be a Boolean, I guess so casting is happening here?

Reply:
I think that the way things currently are...

- At the controller level (in both api/web/push and api/v1/web/push) ... if there's any value at all supplied, it will pass along to model save; if there's no value provided the controller passes in boolean false
- At model level, these rules are applied while persisting - https://api.rubyonrails.org/classes/ActiveModel/Type/Boolean.html